### PR TITLE
Fix pkg/client tests and make them CI Run-able

### PR DESCRIPTION
**Pull request by Oozyx**
## Changes

- Add client test suite 
- Add tests for client address getters

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: pkg/motor/motor.go [hunk 1]
```diff
@@ -274,3 +276,83 @@ func (w *motorNodeImpl) SendTx(routeUrl string, msg sdk.Msg) ([]byte, error) {
 	}
 	return resp.GetTxResponse().Marshal()
 }
+
+func SetupTestAddressWithKeys(motor MotorNode) (error) {
```
Comment (mcjcloud) [inline pkg/motor/motor.go]: I don't think this is a good place for these test-related functions.

File: pkg/motor/motor.go [hunk 2]
```diff
@@ -274,3 +276,83 @@ func (w *motorNodeImpl) SendTx(routeUrl string, msg sdk.Msg) ([]byte, error) {
...
+	name := fmt.Sprintf("./test_keys/%s", n)
+	file, err := os.Create(name)
+	if err != nil {
+		return false
+	}
+	defer file.Close()
+
+	_, err = file.Write(aesKey)
+	return err == nil
+}
```
Comment (mcjcloud) [inline pkg/motor/motor.go]: I notice there are a lot of missing newlines in files. Are you using gofmt?

**Review by mcjcloud — CHANGES_REQUESTED** [review]

File: pkg/client/client_test.go [hunk 3]
```diff
@@ -1,44 +1,78 @@
...
+func (suite *ClientTestSuite) Test_GetFaucetAddress() {
+	env_path := filepath.Join(projectpath.Root, ".env")
+	err := godotenv.Load(env_path)
+  assert.NoError(suite.T(), err, "Load .env success")
+	assert.Equal(suite.T(), os.Getenv("BLOCKCHAIN_FAUCET"), suite.motorNode.GetClient().GetFaucetAddress(), "Faucet address should be the same")
+}
+
+func (suite *ClientTestSuite) Test_GetRPCAddress() {
+	env_path := filepath.Join(projectpath.Root, ".env")
+	err := godotenv.Load(env_path)
```
Comment (joshLong145) [inline pkg/client/client_test.go]: do we have to do this for every test? i think it could be in the setup.

**Review by joshLong145 — CHANGES_REQUESTED** [review]

File: pkg/motor/motor.go (hunk 1 above)
Comment (joshLong145) [reply pkg/motor/motor.go]: agreed should be in `testutils`

File: pkg/client/client_test.go (hunk 3 above)
Comment (Oozyx) [reply pkg/client/client_test.go]: Now it's in the suite setup

File: pkg/motor/motor.go (hunk 2 above)
Comment (Oozyx) [reply pkg/motor/motor.go]: my vscode is set to formatOnSave but for some reason newlines aren't properly handled. I just ran go fmt on the package so should be fine now

File: pkg/motor/motor.go (hunk 1 above)
Comment (Oozyx) [reply pkg/motor/motor.go]: refactored

File: pkg/client/client.go [hunk 4]
```diff
@@ -145,9 +145,9 @@ func (c *Client) GetIPFSApiAddress() string {
 	}
 
 	err = godotenv.Load(env_path)
-  if err != nil {
-    log.Fatal(err)
-  }
+	if err != nil {
+		log.Fatal(err)
```
Comment (joshLong145) [inline pkg/client/client.go:149]: why two tabs on these?

**Review by joshLong145 — APPROVED** [review]
Fix formatting errors with double tabs and its good to merge

**Comment by ntindle** [comment]
Can we fix `app/simulation_test.go` and `testutil/network/network.go` as well in this pr? They are throwing errors on every pr right now and it should be a pretty quick fix

**Comment by mcjcloud** [comment]
Where are the simulation test changes coming from? Are they relevant to this PR?

**Review by mcjcloud — APPROVED** [review]

**Pull request merged**